src/coreclr/jit/emitriscv64.cpp

```diff
@@ -1087,45 +1112,56 @@ void emitter::emitIns_I_la(emitAttr size, regNumber reg, ssize_t imm)
 {
```

Comment:
I updated a `emitIns_I_la` which make a constant value. However, I think there is better solution for `emitIns_I_la`. If you have any idea, please help me! Thank you.

Reply:
It can be optimized later, when the RISC-V codes are ready to go into the optimizing phase.

Reply:
For other platforms we have `instGen_Set_Reg_To_Imm` which is responsible for emitting the instructions to set a register to an arbitrarily sized immediate. However, it lives in `CodeGen`.

I can see that the `emitIns_I_la` is used for some contained addresses with large offsets that need to be first materialized into a register. In general, it would be better to avoid containing these addresses in the first place as it misses the point of containment, which is that the parent node can more efficiently handle the codegen of the child node. If the codegen is going to materialize the value into a register anyway, then there is no reason to mark it contained in the first place.

Reply:
I see. Next time, I will update emitIns_I_la to instGen_Set_Reg_To_Imm and check containment on RISC-V.
Thank you!